### PR TITLE
detect-file-data/v6: add tests for SMTP file data

### PR DESCRIPTION
**Pull request by TheKharleeci**
Test conversion is shown below:

smtp-file-data-01: DetectEngineSMTPFiledataTest01

smtp-file-data-02: DetectEngineSMTPFiledataTest03

Previous PR: #708

Link to redmine ticket: https://redmine.openinfosecfoundation.org/issues/4938

Corresponding Suricata PR: OISF/suricata#6793

### CODE REVIEW

File: tests/smtp-file-data-01/test.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+requires:
+  min-version: 6
```
Comment (inashivb) [inline tests/smtp-file-data-01/test.yaml:2]: Q: Could you please tell why this was added? Did you find a backport missing only in 5?

**Review by inashivb — COMMENTED** [review]
This one needs some attention. ~I'll take a look at this today. Please leave it at this rev so I can check what is going wrong and why does it pass on Alma. :thinking:~
There's one question for you inline :)

File: tests/smtp-file-data-01/test.yaml (hunk 1 above)
Comment (TheKharleeci) [reply tests/smtp-file-data-01/test.yaml:2]: Yes. It without this the CI fails.

Comment (jufajardini) [reply tests/smtp-file-data-01/test.yaml:2]: I suggested that she added it, as even after rebasing the tests were still failing for 5. I'm sorry if that was a misdirection...

**Comment by inashivb** [comment]
Merged w https://github.com/OISF/suricata-verify/pull/753 Thanks! :)

**Pull request closed without merging**